ci/jenkins/test.sh

```diff
@@ -839,6 +853,19 @@ function run_conformance {
     kubectl rollout status daemonset/antrea-agent -n kube-system
 
     set +e
+    # Conformance DNS test cases fail in ipv6 Kind cluster because here CoreDNS pods are IPV6 pods 
+    # and it can't reach IPV4 upstream nameserver.
+    # Note:Default CoreDNS ConfigMap uses /etc/resolv.conf to get upstream nameservers listed in resolv.conf file on the host,
+    # the upstream nameservers are accessed by IPV4, when single stack IPV6 pods try to connect ipv4 nameserver it return servfail,
+    # with the new ConfigMap we will forward DNS query to "vmware.com:53" and it will return 'NXDOMAIN' so that client keep searching.
+    # For more detail: https://github.com/kubernetes/kubernetes/issues/94794 
```

Comment:
Please keep the format consistent event it's a comment.

```suggestion
    # Conformance DNS test cases fail in IPv6 Kind cluster because here CoreDNS pods are IPv6 pods 
    # and it can't reach IPV4 upstream nameserver.
    # Note: Default CoreDNS ConfigMap uses /etc/resolv.conf to get upstream nameservers listed in resolv.conf file on the host,
    # the upstream nameservers are accessed by IPv4, when single stack IPv6 pods try to connect IPv4 nameserver it return servfail,
    # with the new ConfigMap we will forward DNS query to "vmware.com:53" and it will return 'NXDOMAIN' so that client keep searching.
    # For more detail: https://github.com/kubernetes/kubernetes/issues/94794 
```